File: apps/feedback-service-admin/src/api/feedback/FeedbackCreateInput.ts
import { ReviewCreateNestedManyWithoutFeedbacksInput } from "./ReviewCreateNestedManyWithoutFeedbacksInput";

export type FeedbackCreateInput = {
  booking?: string | null;
  comments?: string | null;
  rating?: number | null;
  reviews?: ReviewCreateNestedManyWithoutFeedbacksInput;
};

File: apps/billing-service/src/APIs/Feedback/Dtos/FeedbackCreateInput.cs
namespace BillingService.APIs.Dtos;

public class FeedbackCreateInput
{
    public string? Comments { get; set; }

    public DateTime CreatedAt { get; set; }

    public Customer? Customer { get; set; }

    public string? Id { get; set; }

    public int? Rating { get; set; }

    public DateTime UpdatedAt { get; set; }
}
